build-configs/index.js

```diff
@@ -21,6 +21,15 @@ const loadBuildConfig = (buildConfigName: ?string): BuildConfigType => {
   if (!buildConfig) {
     throw Error(`Invalid BUILD_CONFIG_NAME supplied: ${buildConfigName}`)
   }
+
+  const { featureFlags: { pushNotifications }, android, ios } = buildConfig
+
+  if (android.googleServices === null !== ios.googleServices === null) {
+    console.warn('Google services should be configured for both or no platform!')
+  } else if (pushNotifications === (android.googleServices === null)) {
+    console.warn('Push notification feature flag does not match google service configuration!')
```

Comment:
Probably errors might be better here... Would be even better to check this at compile time (or is this compile time^^).

Reply:
Not sure... If I add just add a new firebase configuration but don't want to have the feature enabled already this is fine imo.